com.krillbite.ThePlan.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: com.krillbite.ThePlan
+runtime: org.freedesktop.Platform
+runtime-version: 1.6
+sdk: org.freedesktop.Sdk
+command: ThePlan.sh
+finish-args:
+- --share=ipc
+- --socket=x11
+- --device=dri
+- --socket=pulseaudio
+- --filesystem=~/.config/unity3d:create
+
+modules:
+- shared-modules/glu/glu-9.0.0.json
+
+- name: ThePlan
+  buildsystem: simple
+  build-commands:
+  - cp -R 'The Plan_Data' '/app/The Plan_Data'
+  - install -Dm755 'The Plan.x86' '/app/The Plan.x86'
+  - install -Dm755 'The Plan.x86_64' '/app/The Plan.x86_64'
+  - install -Dm755 launcher.sh /app/bin/ThePlan.sh
+  - install -Dm644 com.krillbite.ThePlan.appdata.xml /app/share/appdata/com.krillbite.ThePlan.appdata.xml
+  - install -Dm644 com.krillbite.ThePlan.desktop /app/share/applications/com.krillbite.ThePlan.desktop
+  - install -Dm644 icon32x32.png /app/share/icons/hicolor/32x32/apps/com.krillbite.ThePlan.png
+  - install -Dm644 icon64x64.png /app/share/icons/hicolor/64x64/apps/com.krillbite.ThePlan.png
+  - install -Dm644 icon128x128.png /app/share/icons/hicolor/128x128/apps/com.krillbite.ThePlan.png
+
+  sources:
+  - type: archive
```

Comment:
This is proprietary so we can't redistribute it, you need an `extra-data` source type, see other proprietary apps like Discord.